components/Layout.tsx

```diff
@@ -61,41 +64,96 @@ export default function Layout({
   children: React.ReactNode;
 }) {
   const [mobileOpen, setMobileOpen] = useState(false);
+  const [searchIsOpen, setSearchOpen] = useState(false);
+  const [searchTerm, setSearchTerm] = useState('');
 
   const { width } = useWindowSize();
-  const { asPath } = useRouter();
+  const router = useRouter();
+  const asPath = router.asPath;
 
   const handleDrawerToggle = () => {
     setMobileOpen(!mobileOpen);
   };
 
   const drawer = (
-    <List>
-      {[...navItems]
-        .sort((a, b) => {
-          if (a.title === 'Home') {
-            return -1;
-          }
-          if (b.title === 'Home') {
-            return 1;
-          }
-          return a.title.toLowerCase() > b.title.toLowerCase() ? 1 : -1;
-        })
-        .map(({ title: itemTitle, href, Icon }) => (
+    <List id='nav-sidebar'>
+      {[
+        {
+          title: 'Home',
+          href: '/',
+          Icon: HomeIcon,
+        },
+        {
+          title: 'Search',
```

Comment:
I think it would be better UX to have search at the top, but just show the search field instead of hiding it behind a button. That way we can hit the tab key twice and start typing a search query

Reply:
Sounds good! Wasn't sure how desirable it would be to have all the nav items consistent in the sidebar. I'll refactor that